### PR TITLE
update ephemeral attendance page

### PR DESCRIPTION
**Pull request by altsalt**

### CODE REVIEW

**Review by strugee — APPROVED** [review]
LGTM ~assuming that you didn't update the button on purpose since it still points to the test~ mixed up PRs, sorry

File: attend/ephemeral.md [hunk 1]
```diff
@@ -8,37 +8,37 @@ description: How to attend SeaGL 2022 with an ephemeral Matrix account
 
 # Attending SeaGL
 
-SeaGL uses a chat system called [Matrix] to run our virtual conference. We've got things set up so Matrix is super easy to use for you - just follow these 3 steps. But if you _do_ run into any trouble, you can email <matrix-help@seagl.org> and we'll help you get it sorted out. TODO - possibly link to Kiwi IRC too.
+SeaGL's virtual conference space exists in [Matrix], an open, decentralized chat network. We've set things up so that using Matrix should be easy—just follow these 3 steps. If you _do_ run into any trouble, email <matrix-help@seagl.org> and we'll help you get it sorted out ASAP. <!-- TODO - possibly link to Kiwi IRC too. -->
```
Comment (AndrewKvalheim) [inline attend/ephemeral.md:11]: Broken link?

File: attend/ephemeral.md [hunk 2]
```diff
@@ -8,37 +8,37 @@ description: How to attend SeaGL 2022 with an ephemeral Matrix account
...
+3. While your account exists, it will function as a full Matrix account. Consider joining the Matrix room of a free software project you hear about at the conference! <!-- TODO it would be rad if we could actually see how many talks are about projects with Matrix rooms -->
+4. If you decide you like Matrix, we encourage you to get a permanent account and hang out in the [SeaGL space](https://matrix.to/#/#SeaGL:seagl.org) year-round! There are lots of [homeserver options](https://joinmatrix.org/servers/) for using Matrix, but we often recommend going with [Matrix*.org*](https://matrix.org/faq/#who-and-how) as a service provider—a public service operated by the foundation's non-profit.
+5. There are [many clients](https://matrix.org/docs/projects/try-matrix-now) that you can use to access Matrix, but we often recommend going with [Element](https://app.element.io/) since it's open source and available on many platforms, including the web.
```
Comment (strugee) [inline attend/ephemeral.md:37]: Nit: I like that we're separating stuff out into clearer bullet points, but I think this needs an "if you decide you like Matrix" type message too. I want folks to understand what they're ok to ignore until they're more comfortable, to not overwhelm them

File: attend/ephemeral.md [hunk 3]
```diff
@@ -8,37 +8,37 @@ description: How to attend SeaGL 2022 with an ephemeral Matrix account
...
-## Step 3 - have fun at SeaGL!
 
-We hope you enjoy hanging out with our flock this year. Our friendly Matrix bot, Patch, will help guide you around the conference, and if you have any questions you can either ask in the "Welcome" room or click here (TODO make this a link) to go directly there.
+## Step 3 - have fun at SeaGL!
 
-As a reminder, participation in SeaGL is subject to our [Code of Conduct](/code_of_conduct).
+We hope you enjoy hanging out with the flock this year. Our friendly Matrix bot, Patch, will help guide you around the conference. If you have any questions, please bring them up in the ["Welcome"](https://matrix.to/#/#2022-welcome:seagl.org) or ["Info Booth"](https://matrix.to/#/#2022-info-booth:seagl.org) rooms.
 
-[Element]: https://element.io/
-[Matrix]: https://matrix.org/
```
Comment (strugee) [inline attend/ephemeral.md:42]: These shouldn't be removed - maybe this was what https://github.com/SeaGL/seagl.github.io/pull/365#discussion_r1010960695 was referring to?

File: attend/ephemeral.md [hunk 4]
```diff
@@ -8,37 +8,37 @@ description: How to attend SeaGL 2022 with an ephemeral Matrix account
...
+5. There are [many clients](https://matrix.org/docs/projects/try-matrix-now) that you can use to access Matrix, but we often recommend going with [Element](https://app.element.io/) since it's open source and available on many platforms, including the web.
 
-## Step 3 - have fun at SeaGL!
 
-We hope you enjoy hanging out with our flock this year. Our friendly Matrix bot, Patch, will help guide you around the conference, and if you have any questions you can either ask in the "Welcome" room or click here (TODO make this a link) to go directly there.
+## Step 3 - have fun at SeaGL!
 
-As a reminder, participation in SeaGL is subject to our [Code of Conduct](/code_of_conduct).
+We hope you enjoy hanging out with the flock this year. Our friendly Matrix bot, Patch, will help guide you around the conference. If you have any questions, please bring them up in the ["Welcome"](https://matrix.to/#/#2022-welcome:seagl.org) or ["Info Booth"](https://matrix.to/#/#2022-info-booth:seagl.org) rooms.
```
Comment (strugee) [inline attend/ephemeral.md:42]: The matrix.to links are wrong here - these folks should be linked to the attend portal

**Review by strugee — COMMENTED** [review]
Missed some stuff since the GitHub view I clicked into only showed me part of the diff - sorry!

I'm happy to make changes here if you don't have the bandwidth. I know I'm nitpicking a little, but I _really_ want to get this page especially right since this is the absolute first thing folks are going to interact with

File: attend/ephemeral.md (hunk 3 above)
Comment (altsalt) [reply attend/ephemeral.md:42]: I generally go with in-line links unless it is being repeated throughout or super unwieldy.

File: attend/ephemeral.md (hunk 4 above)
Comment (altsalt) [reply attend/ephemeral.md:42]: Okay, I think these can be set as redirects within the attend portal.

**Comment by altsalt** [comment]
@strugee please make the changes. I'm not tied to anything here, just went on a cleaning/editing pass as it seemed needed.

File: attend/ephemeral.md (hunk 3 above)
Comment (strugee) [reply attend/ephemeral.md:42]: Lol same, it was like this cause I stole it directly from another page 😂

File: attend/ephemeral.md (hunk 4 above)
Comment (strugee) [reply attend/ephemeral.md:42]: @AndrewKvalheim can Element already deep link to a room or is this going to need engineering from you?

Comment (altsalt) [reply attend/ephemeral.md:42]: Something like this should work:
`https://attend.seagl.org/#/room/#2022-info-booth:seagl.org`

**Comment by strugee** [comment]
Addressed in https://github.com/SeaGL/seagl.github.io/pull/372

**Pull request closed without merging**